src/tests/system/tests/test_netgroups.py

```diff
@@ -241,3 +243,131 @@ def test_netgroup__lookup_nested_groups_with_host_and_domain_values_present(
     result = client.tools.getent.netgroup("nested_group")
     assert result is not None
     assert expected in result.members
+
+
+@pytest.mark.importance("low")
+@pytest.mark.ticket(bz=802207)
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_netgroup__fully_qualified_names(client: Client, provider: GenericProvider):
+    """
+    :title: SSSD hangs when use fully qualified names set to true
+    :setup:
+        1. Configures the SSSD domain to use fully qualified names
+        2. User with the name "user-1" is created
+        3. A netgroup named "ng-1" is created and the previously added user "user-1" is added to this netgroup
```

Comment:
Mentioning `previously` here makes it difficult to understand. I'd just mention that `user-1` is added to this netgroup